lib/components/AlbumScreen/track_list_tile.dart

```diff
@@ -205,18 +226,38 @@ class TrackListTile extends StatelessWidget {
               id: parentItem?.id.raw ?? "",
               item: parentItem,
             )));
+          break;
+        case ItemSwipeActions.nothing:
+          break;
       }
 
-      GlobalSnackbar.message(
-        (scaffold) => FinampSettingsHelper.finampSettings.swipeInsertQueueNext
-            ? AppLocalizations.of(scaffold)!.confirmAddToNextUp("track")
-            : AppLocalizations.of(scaffold)!.confirmAddToQueue("track"),
-        isConfirmation: true,
-      );
+      if (followUpAction != ItemSwipeActions.nothing) {
+        GlobalSnackbar.message(
+          (scaffold) => (followUpAction == ItemSwipeActions.addToNextUp)
+              ? AppLocalizations.of(scaffold)!.confirmAddToNextUp("track")
+              : (followUpAction == ItemSwipeActions.playNext)
+                  ? AppLocalizations.of(scaffold)!.confirmPlayNext("track")
+                  : AppLocalizations.of(scaffold)!.confirmAddToQueue("track"),
+          isConfirmation: true,
+        );
+      }
 
       return false;
     }
 
+    String getSwipeActionText(BuildContext context, ItemSwipeActions action) {
+      switch (action) {
+        case ItemSwipeActions.addToQueue:
+          return AppLocalizations.of(context)!.addToQueue;
+        case ItemSwipeActions.addToNextUp:
+          return AppLocalizations.of(context)!.addToNextUp;
+        case ItemSwipeActions.playNext:
+          return AppLocalizations.of(context)!.playNext;
+        default:
+          return "";
+      }
+    }
```

Comment:
This is normally handled in `finamp_models.dart`. There should be a `getLocalizedString(BuildContext context)` method on the enum type.